index.js

```diff
@@ -104,21 +104,22 @@ Limiter.prototype.get = function (fn) {
     }
 
     db.multi()
-      .set([count, n - 1, 'PX', ex * 1000 - dateNow, 'XX'])
+      .decr(count)
+      .pexpire([count, ex * 1000 - dateNow])
       .pexpire([limit, ex * 1000 - dateNow])
       .pexpire([reset, ex * 1000 - dateNow])
-      .exec(function (err, res) {
+      .exec(function(err, res) {
         if (err) return fn(err);
         if (isFirstReplyNull(res)) return mget();
-        n = n - 1;
+        n = Array.isArray(res[0]) ? ~~res[0][1] : ~~res[0];
```

Comment:
If it is `redis` module, then `res[0]` is a string value. If it's `ioredis` then it's an array.